skins/oasis/css/core/page.scss

```diff
@@ -281,6 +279,60 @@
 	}
 }
 
+#WikiaRecentActivity {
```

Comment:
id selectors are hard to override. Is it possible to convert it to class selector?